<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BugDetectClass</class>
 <widget class="QWidget" name="BugDetectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BugDetect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="pGraphicsViewer" name="graphicsView"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>0.原图</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>1.灰度图</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>3.梯度图</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>2.降噪</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>4.再降噪</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>5.形态学</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string>6.细节刻画</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_8">
     <property name="text">
      <string>7.查找轮廓</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_9">
     <property name="text">
      <string>8.绘制轮廓</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_10">
     <property name="text">
      <string>9.裁剪</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>pGraphicsViewer</class>
   <extends>QGraphicsView</extends>
   <header>../GraphicsViewer/pGraphicsViewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="BugDetect.qrc"/>
 </resources>
 <connections/>
</ui>
